.github/workflows/ci.yml

```
@@ -72,4 +60,4 @@ jobs:
         run: |
           echo "### ✅ Allure Report" >> $GITHUB_STEP_SUMMARY
           echo "" >> $GITHUB_STEP_SUMMARY
-          echo "[Открыть отчёт](https://${{ github.repository_owner }}.github.io/${{ github.event.repository.name }}/)" >> $GITHUB_STEP_SUMMARY
+          echo "[Открыть отчёт](https://${{ github.repository_owner }}.github.io/${{ github.event.repository.name }}/)" >> $GITHUB_STEP_SUMMARY
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)